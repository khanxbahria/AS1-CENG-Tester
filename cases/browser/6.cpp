#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("nn", "eee","nn");
Tab B = Tab("ll", "k","iii");
Tab C = Tab("zz", "jjj","a");
Tab D = Tab("jjj", "pp","zz");
Tab E = Tab("vv", "ll","f");
Tab F = Tab("m", "bbb","nn");
Tab G = Tab("ddd", "ccc","nn");
Tab H = Tab("qqq", "aaa","a");
Tab I = Tab("m", "pp","ii");
Tab J = Tab("jj", "uuu","p");
Tab K = Tab("a", "aa","hh");
Tab L = Tab("nnn", "g","zzz");
Tab M = Tab("i", "vv","hh");
Tab N = Tab("ccc", "rrr","nn");
Tab O = Tab("uu", "m","qqq");
Tab P = Tab("f", "k","l");
Tab Q = Tab("aaa", "qq","pp");
Tab R = Tab("ccc", "f","h");
Tab S = Tab("rrr", "rr","qqq");
Tab T = Tab("t", "l","yy");
Tab U = Tab("sss", "kkk","d");
Tab V = Tab("ii", "n","sss");
Tab W = Tab("c", "v","vvv");
Tab X = Tab("jj", "nn","w");
Tab Y = Tab("v", "cc","z");
Tab Z = Tab("j", "ww","bbb");
a.changeActiveTabTo(3);
b.newTab(A);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
c.newTab(B);
d.changeActiveTabTo(5);
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.newTab(C);
a.newTab(D);
b.newTab(E);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
a.newTab(F);
b.changeActiveTabTo(4);
c.moveActiveTabTo(4);
d.newTab(G);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
c.changeActiveTabTo(5);
d.newTab(H);
a.newTab(I);
b.moveActiveTabTo(1);
c.newTab(J);
d.newTab(K);
a.changeActiveTabTo(4);
b.newTab(L);
c.moveActiveTabTo(4);
d.newTab(M);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.newTab(N);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.changeActiveTabTo(4);
b.newTab(O);
c.newTab(P);
d.changeActiveTabTo(4);
a.newTab(Q);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.moveActiveTabTo(2);
a.changeActiveTabTo(0);
b.newTab(R);
c.newTab(S);
d.newTab(T);
a.moveActiveTabTo(1);
b.changeActiveTabTo(1);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.newTab(U);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
a.moveActiveTabTo(4);
b.moveActiveTabTo(4);
c.newTab(V);
d.changeActiveTabTo(4);
a.newTab(W);
b.newTab(X);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.changeActiveTabTo(3);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
d.newTab(Y);
a.moveActiveTabTo(2);
b.newTab(Z);
c.moveActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
d.moveActiveTabTo(3);
b.moveActiveTabTo(4);
c.changeActiveTabTo(4);
a.moveActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
b.changeActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
a.moveActiveTabTo(3);
b.changeActiveTabTo(3);
c.moveActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
a.changeActiveTabTo(3);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.changeActiveTabTo(5);
d.changeActiveTabTo(5);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
d.changeActiveTabTo(4);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(3);
c.changeActiveTabTo(3);
a.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
b.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
a.moveActiveTabTo(5);
b.moveActiveTabTo(5);
c.moveActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(2);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
a.changeActiveTabTo(5);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->closeWindow();
brows->closeWindow();
brows->closeWindow();
brows->closeEmptyWindows();
brows->getWindow(6);
brows->getWindow(1);
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->switchToWindow(6);
brows->closeWindow();
brows->closeEmptyWindows();
brows->getWindow(4);
brows->closeEmptyWindows();
brows->closeWindow();
brows->mergeAllWindows();
brows->newWindow();
brows->print();
brows->getWindow(5);
brows->print();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->getWindow(5);
brows->closeEmptyWindows();
brows->closeWindow();
brows->mergeAllWindows();
brows->print();
brows->newWindow();
brows->closeWindow();
brows->switchToWindow(6);
brows->newWindow();
brows->switchToWindow(5);
brows->print();
brows->print();
brows->newWindow();
brows->mergeAllWindows();
brows->getWindow(8);
brows->closeEmptyWindows();
brows->getWindow(6);
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->print();
brows->mergeAllWindows();
brows->print();
brows->getWindow(8);
brows->switchToWindow(9);
brows->print();
brows->print();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->switchToWindow(9);
brows->newWindow();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->getWindow(4);
brows->getWindow(1);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->switchToWindow(7);
brows->mergeAllWindows();
brows->newWindow();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->print();
brows->newWindow();
brows->switchToWindow(9);
brows->switchToWindow(0);
brows->closeEmptyWindows();
brows->switchToWindow(5);
brows->switchToWindow(2);
brows->mergeAllWindows();
brows->print();
brows->mergeAllWindows();
brows->print();
brows->closeEmptyWindows();
brows->switchToWindow(6);
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->switchToWindow(4);
brows->getWindow(7);
brows->closeWindow();
brows->newWindow();
brows->switchToWindow(7);
brows->getWindow(4);
brows->getWindow(3);
brows->mergeAllWindows();
brows->switchToWindow(5);
brows->print();
brows->mergeAllWindows();
brows->print();
brows->getWindow(7);
brows->getWindow(4);
brows->mergeAllWindows();
brows->switchToWindow(7);
brows->print();
brows->print();
brows->print();
brows->closeWindow();
brows->closeWindow();
brows->print();
brows->mergeAllWindows();
brows->getWindow(4);
brows->getWindow(5);
brows->closeWindow();
brows->closeWindow();
brows->closeWindow();
brows->getWindow(4);
brows->mergeAllWindows();
brows->closeWindow();
brows->switchToWindow(7);
brows->switchToWindow(5);
brows->switchToWindow(8);
brows->newWindow();
brows->getWindow(4);
brows->newWindow();
brows->switchToWindow(8);
brows->mergeAllWindows();
brows->closeEmptyWindows();
delete brows;
}
	return 0;
}