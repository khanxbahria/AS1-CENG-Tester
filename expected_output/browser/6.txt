The active tab is tab 0




The active tab is tab 0




The browser is empty
The active tab is tab 0




The active tab is tab 0




The active tab is tab 1








The active tab is tab 1








The active tab is tab 1








The active tab is tab 1








The active tab is tab 4




















The active tab is tab 5
























Window is empty
The list is empty.
The active tab is tab 5
























Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The browser is empty
